models/location-sql.js

```diff
@@ -0,0 +1,58 @@
+import db from '../db/connection.js'
+
+// Get all location data
+export async function getAllLocationData() {
+    const result = await db.query(`SELECT * FROM location;`);
+    return result.rows;
+}
+
+// Get location data by loc_id
+export async function getLocationByLocID(loc_id) {
+    const result = await db.query(`SELECT * FROM location WHERE loc_id = $1;`, [loc_id])
+    return result.rows;
+}
+
+// Get location data by media_id
+export async function getLocationByMedID(media_id) {
+    const result = await db.query(`SELECT * FROM location WHERE media_id = $1;`, [media_id])
+    return result.rows;
+}
+
+// Get location by lat lon
+export async function getLocationByLatLon(latitude, longitude) {
+    const result = await db.query(`SELECT * FROM location WHERE latitude = $1 AND longitude = $2;`, [latitude, longitude])
```

Comment:
- change latitude/longitude to lat/lng
- revisit whether we need this query or if retrieving info based on loc_id is enough (not sure if it will work; brain died an hour ago)